electron/src/mods.ts

```diff
@@ -68,7 +184,32 @@ export class ModsHandler {
 
         return directory
             .filter(entry => entry.name.endsWith(modFileSuffix))
-            .filter(entry => !entry.isDirectory())
             .map(entry => path.join(entry.path, entry.name));
     }
+
+    private async readDirectory(dirPath: string): Promise<DirectoryNode> {
+        const entries = await fs.readdir(dirPath, {
+            withFileTypes: true,
+        });
+        const contents: Record<string, Node> = {};
+
+
+        for (const entry of entries) {
+            contents[entry.name] = entry.isFile() ? {
```

Comment:
maybe use `!entry.isDirectory()` like before, for funny things like symlinks. (though either way this can throw for e.g. symlinks to files/directories, I think)